bokehjs/src/lib/models/mappers/continuous_color_mapper.ts

```diff
@@ -33,15 +33,34 @@ export abstract class ContinuousColorMapper extends ColorMapper {
     })
   }
 
+  protected abstract scan<T>(data: Arrayable<number>, palette: Arrayable<T>) : any
+
+
+  protected _v_compute<T>(data: Arrayable<number>, values: Arrayable<T>,
+    palette: Arrayable<T>, colors: {nan_color: T, low_color?: T, high_color?: T}): void {
+
+    const {nan_color, low_color, high_color} = colors
+    const scan_data = this.scan(data, palette)
+
+    for (let i = 0, end = data.length; i < end; i++) {
+      const d = data[i]
+
+      if (isNaN(d)) {
+        values[i] = nan_color
+        continue
+      }
+        values[i] = this.cmap(d, palette, low_color, high_color, scan_data)
```

Comment:
indentation here is off, but with only two 1-line branches I would prefer an `if-else` rather than the `continue`

Reply:
Addressed in e008d63